README.md

```diff
@@ -1,5 +1,18 @@
-# license-list-XML
-This is a workspace for developing and implementing a new XML source format for the SPDX License List. This source format is (or will be) used to generate the authoritative file formats for licenses on the SPDX License List. These authoritative [target/destination formats](https://github.com/spdx/license-list-data) will remain the same for the time being.
+# SPDX License List Source Files
+The [SPDX License List](https://spdx.org/licenses/) is a list of common open-source licenses (and associated exceptions) used to easily and efficiently identify such licenses in SPDX documents, SPDX identifiers (e.g. in source code), package managers, or elsewhere.
+This repository contains the XML "source" files (and schema) used to generate the authoritative, supported SPDX list file formats. 
 
-# NOT STABLE OR AUTHORITATIVE
-The files in this repo are not stable in XML structure or in substance (license text). Although they are being worked on publicly and transparently here on github, they are currently only suitable for internal use by the SPDX Legal Team and the SPDX Technical Team. Please do _not_ rely on these files just yet, and please continue to use the resources on [spdx.org](https://spdx.org/) and [plain license-list repo](https://github.com/spdx/license-list) instead when developing tools or ingesting license text.
+# This Repository: Not Stable or Authoritative
```

Comment:
I'd drop "Authoritative", since this repo will be authoritative.  And to tighten up the unstable wording, how about:

> ## This repository is unstable

Reply:
agree